### PR TITLE
Add postgres-client and pg-activity

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
@sbidoul

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,16 @@ RUN apt-get update && apt-get install -y \
...
     jq \
-  && rm -fr /var/lib/apt/lists/*
+    python3-pip \
+    python3-psycopg2
+
+# install postgres client and pg-activity
+
+RUN apt -y install --no-install-recommends postgresql-client \
+  && apt -y clean \
+  && rm -rf /var/lib/apt/lists/*
```
Comment (sbidoul) [inline Dockerfile]: Merge all this with the previous RUN command (which miss the cleanup, btw)?

**Comment by sbidoul** [comment]
@adrienpeiffer can you also update the README?

**Comment by adrienpeiffer** [comment]
Done

File: Dockerfile [hunk 2]
```diff
@@ -23,7 +25,12 @@ RUN apt-get update && apt-get install -y \
     zip \
     unzip \
     jq \
-  && rm -fr /var/lib/apt/lists/*
+    python3-pip \
+    python3-psycopg2 \
+    && apt -y install --no-install-recommends postgresql-client \
+    && apt -y clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && pip install pg-activity
```
Comment (sbidoul) [inline Dockerfile]:
```suggestion
    && python3 -m pip install pg-activity
```

**Pull request merged**